internal/cli/performanceadvisor/namespaces/list.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package namespaces
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const listTemplate = `NAMESPACE	TYPE{{range .Namespaces}}
+{{.Namespace}}	{{.Type}}{{end}}
+`
+
+type ListOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	store       store.PerformanceAdvisorNamespacesLister
+	processName string
+	hostID      string
+	host        string
+	since       int64
+	duration    int64
+}
+
+func (opts *ListOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+func (opts *ListOpts) Run() error {
+	r, err := opts.store.PerformanceAdvisorNamespaces(opts.ConfigProjectID(), opts.host, opts.newNamespaceOptions())
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+func (opts *ListOpts) newNamespaceOptions() *atlas.NamespaceOptions {
+	return &atlas.NamespaceOptions{
+		Since:    opts.since,
+		Duration: opts.duration,
+	}
+}
+
+func (opts *ListOpts) validateProcessName() error {
+	length := 2
```

Comment:
[nit] the reason there's a linter that complains about magic numbers is that we should be clear with descriptions on what they mean
```suggestion
	const expectedLength = 2
```